enlighten/StatusIndicators.py

```diff
@@ -144,6 +144,14 @@ def update_visibility(self):
                         lamp = "disconnected"
                         lamp_tt = f"laser disarmed (cannot fire)"
 
+            # default behavior should be to do the normal processing
+            # Multispec case here checks all and will overwrite if a laser is on
+            all_specs = self.multispec.get_spectrometers()
+            specs_lasers_on = [s.settings.state.laser_enabled for s in all_specs]
+            if any(specs_lasers_on):
+                lamp = "warning"
+                lamp_tt = "laser is firing"
```

Comment:
Technically this is un-doing some of the laser interlock monitoring from the block above. The new code is based on ENLIGHTEN's SpectrometerState.laser_enabled, which actually indicates whether the laser has been ASKED to fire.  If the key-switch on the back of the spectrometer has been turned, or if the interlock continuity circuit has been broken, then the laser won't actually be firing, even though state.laser_enabled is True.

However, not many released customer systems yet support laser interlock monitoring, and I agree that this is an improvement over the current standard case.